ED/protocol.html

```diff
@@ -1047,7 +1047,7 @@ <h3 property="schema:name">Deleting Resources</h3>
               <section id="resource-representations" inlist="" rel="schema:hasPart" resource="#resource-representations">
                 <h3 property="schema:name">Resource Representations</h3>
                 <div datatype="rdf:HTML" property="schema:description">
-                  <p><span about="" id="server-representation-turtle-jsonld" rel="spec:requirement" resource="#server-representation-turtle-jsonld"><span property="spec:statement">When a <span rel="spec:requirementSubject" resource="#Server">server</span> creates a resource on HTTP <code>PUT</code>, <code>POST</code> or <code>PATCH</code> requests such that the request’s representation data encodes an <em>RDF document</em> [<cite><a class="bibref" href="#bib-rdf11-concepts">RDF11-CONCEPTS</a></cite>] (as determined by the <code>Content-Type</code> header), the server <span rel="spec:requirementLevel" resource="spec:MUST">MUST</span> accept <code>GET</code> requests on this resource when the value of the <code>Accept</code> header requests a representation in <code>text/turtle</code> or <code>application/ld+json</code> [<cite><a class="bibref" href="#bib-turtle">Turtle</a></cite>] [<cite><a class="bibref" href="#bib-json-ld11">JSON-LD11</a></cite>].</span></span> [<a href="https://github.com/solid/specification/issues/45" rel="cito:citesAsSourceDocument">Source</a>] <a href="https://github.com/solid/specification/issues/69" rel="cito:citesAsSourceDocument">Source</a>] [<a href="https://github.com/solid/specification/issues/109" rel="cito:citesAsSourceDocument">Source</a>] [<a href="https://github.com/solid/specification/issues/195" rel="cito:citesAsSourceDocument">Source</a>]</p>
+                  <p><span about="" id="server-representation-turtle-jsonld" rel="spec:requirement" resource="#server-representation-turtle-jsonld"><span property="spec:statement">When a <span rel="spec:requirementSubject" resource="#Server">server</span> creates an <em>RDF source</em> [<cite><a class="bibref" href="#bib-rdf11-concepts">RDF11-CONCEPTS</a></cite>] on HTTP <code>PUT</code>, <code>POST</code> or <code>PATCH</code> requests, the server <span rel="spec:requirementLevel" resource="spec:MUST">MUST</span> accept <code>GET</code> requests on this resource when the value of the <code>Accept</code> header requests a representation in <code>text/turtle</code> or <code>application/ld+json</code> [<cite><a class="bibref" href="#bib-turtle">Turtle</a></cite>] [<cite><a class="bibref" href="#bib-json-ld11">JSON-LD11</a></cite>].</span></span> [<a href="https://github.com/solid/specification/issues/45" rel="cito:citesAsSourceDocument">Source</a>] <a href="https://github.com/solid/specification/issues/69" rel="cito:citesAsSourceDocument">Source</a>] [<a href="https://github.com/solid/specification/issues/109" rel="cito:citesAsSourceDocument">Source</a>] [<a href="https://github.com/solid/specification/issues/195" rel="cito:citesAsSourceDocument">Source</a>]</p>
```

Comment:
I *think* this means to say that GET requests for Turtle or JSON-LD must be _satisfied_ (i.e., the preferred media type must be delivered), not just that the GET requests must be "_accepted_" (whatever that would mean in HTTP). (Also, I added a missing comma.) 

(GitHub is having all sorts of issues with highlighting requested changes; I hope you have a handy DIFF tool.)
```suggestion
                  <p><span about="" id="server-representation-turtle-jsonld" rel="spec:requirement" resource="#server-representation-turtle-jsonld"><span property="spec:statement">When a <span rel="spec:requirementSubject" resource="#Server">server</span> creates an <em>RDF source</em> [<cite><a class="bibref" href="#bib-rdf11-concepts">RDF11-CONCEPTS</a></cite>] on HTTP <code>PUT</code>, <code>POST</code>, or <code>PATCH</code> requests, the server <span rel="spec:requirementLevel" resource="spec:MUST">MUST</span> satisfy <code>GET</code> requests on this resource when the value of the <code>Accept</code> header requests a representation in <code>text/turtle</code> or <code>application/ld+json</code> [<cite><a class="bibref" href="#bib-turtle">Turtle</a></cite>] [<cite><a class="bibref" href="#bib-json-ld11">JSON-LD11</a></cite>].</span></span> [<a href="https://github.com/solid/specification/issues/45" rel="cito:citesAsSourceDocument">Source</a>] <a href="https://github.com/solid/specification/issues/69" rel="cito:citesAsSourceDocument">Source</a>] [<a href="https://github.com/solid/specification/issues/109" rel="cito:citesAsSourceDocument">Source</a>] [<a href="https://github.com/solid/specification/issues/195" rel="cito:citesAsSourceDocument">Source</a>]</p>
```

Reply:
"Accept" as in 2xx-4xx with the exception of 406 "Not Acceptable" ( https://www.rfc-editor.org/rfc/rfc9110#status.406 ).

"Satisfy" is not jumping out to me as much as "accept" (which I find to be concrete). I'd like to look into splitting this requirement up any way. Can we live with "accept" for now or want to press on "satisfy"? :)

Reply:
"Satisfy" as in, the server must provide a representation of the RDF source serialized as the media type in the `Accept:` header, i.e., `text/turtle` or `application/ld+json` — regardless of whether that RDF source is stored by the Solid server as RDF in a triple/quad store, or as *any* RDF serialization.

Your 2xx-4xx "accept" is redundant to the HTTP spec, which mandates such codes be returned for GET on *any* requested resource, regardless of `Accept:` header, *including* non-existent resources (404 is neither your excluded 405 nor 406).

It seems like something's missing from this picture.

Reply:
I mentioned 405 and 406 in my comment because they seemed most relevant. We know the resource is created, and so unless there are arbitrary exceptions or changes to the resource state, it is not going to be a 404. What's perhaps most significant is 406 because the requirement aimed to make sure that when a server creates an RDF source, it should make it available in Turtle and JSON-LD. It doesn't take much to come up with other arguments - not particularly worth the time to discuss (IMO) - such as "but Sarvie, if I only have append access on the container and I POST to the container, then a GET on the resource would be 403..." As true as that may be, it is irrelevant to the point in the requirement.

I'm okay to adopt your suggestion with "satisfy" but not because I find it particularly better than "accept". It is stepping into language-bikeshedding. "fulfill", "accommodate", "handle", "process", and other terms... could work too (possibly with some adjustments).

Reply:
Some language tweaks are definitely bike shedding. I don't think this one is.

To hopefully be clear(er), I would also be OK with "fulfill", "accommodate", "handle", "process", and (probably) other terms, instead of `satisfy`. "Accept" just doesn't convey the same intent as the rest of these, imho.

Reply:
I simply borrowed from 406 Not **Accept**able because the point of the requirement was to have the server "accept" GET w/ particular media types (in Accept header) for what's originally acknowledged as an RDF source. If that doesn't convey what's intended to you (or others), that's fine, I will accept the suggested alternative.